console/src/telemetryFilter.ts

```diff
@@ -34,25 +34,25 @@ const dataHandler = path => {
   );
 };
 
-const apiExplorerHandler = () => {
+const apiExplorerHandler = (): string => {
```

Comment:
For this string constant, we could do:
```ts
const API_EXPLORER_PATH = '/api-explorer' as const;
```
and omit providing return types for the function, so that we would have following functions signature: 

<img width="495" alt="Screenshot 2020-05-04 at 23 46 07" src="https://user-images.githubusercontent.com/9019397/81016968-bc479080-8e61-11ea-8573-f03b02513642.png">